doc/src/programmers-guide.md

```diff
@@ -612,6 +612,9 @@ For more information about Erlang external term format, consult the [Erlang Docu
 
 You can obtain system information about the AtomVM virtual machine via the [`erlang:system_info/1`](./apidocs/erlang/estdlib/erlang.md#system_info1) function, which takes an atom parameter designating the desired datum.  Allowable parameters include
 
+* `os_type` The operating system or runtime environment.
+* `os_version` The sematic version of the currently running operating system or runtime environment.
+* `system_architecture` The processor architecture and OS release (as a binary string).
```

Comment:
This didn't make it :-)

Reply:
Right, I didn't add that for Pico, I just noticed it was completely missing from the docs and didn't want to forget to add it, but Pico support for this key would be good to add.

Reply:
I realized this is currently only implemented on `generic_unix`, so I removed the entry (It should probably have been added in a separate PR anyway). The edocs incorrectly state it is supported on **all** systems, I will put in a separate PR to correct that to list it as only supported on `generic_unix`, until support has been added for other platforms.